virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc

```diff
@@ -50,20 +41,20 @@ Configuring IP address management (IPAM) by specifying the `spec.config.ipam.sub
 
 include::modules/virt-creating-layer2-nad-cli.adoc[leveloffset=+2]
 
-ifndef::openshift-rosa,openshift-dedicated,openshift-rosa-hcp[]
-include::modules/virt-creating-localnet-nad-cli.adoc[leveloffset=+2]
-endif::openshift-rosa,openshift-dedicated,openshift-rosa-hcp[]
+//ifndef::openshift-rosa,openshift-dedicated,openshift-rosa-hcp[]
```

Comment:
Confirm with the service delivery team that this content belongs there, please

Reply:
It doesn't